api-docs/release-notes/releases/cdbv3.10.0.rst

```diff
@@ -0,0 +1,43 @@
+.. version-3.10.0-release-notes:
+
+v3.10.0, October 27, 2020
+-------------------------
+
+What's new
+~~~~~~~~~~
+
+-  Upgrading a single instance will put both the instance and it's upgraded
+   replica into a VERIFY_UPGRADE state.
+
+   **Note:** Both the instance and upgraded replica will remain in this state
+   until the upgraded replica is detached from the instance.
+
+   This is done to keep data current in the upgraded replica while the
+   customer validates that their application doesn't have any issues with the
+   upgraded replica and plan a maintenance to disconnect their application from
```

Comment:
upgraded replica and plans a maintenance to disconnect their application from     [consistent verb]